enterprise/setup/index.md

```diff
@@ -0,0 +1,323 @@
+---
+title: 'Enterprise Edition: Initial Setup'
+backlink_href: /docs/enterprise
+backlink_title: 'pganalyze Enterprise Edition'
+---
+
+These are the installation instructions for [pganalyze Enterprise Edition](/enterprise-postgres-monitoring), which runs fully-contained within your own environment.
+
+The pganalyze Docker image is mostly self-contained, and runs various services inside the Docker container.
+
+```toc
+```
+
+## Installation steps
+
+### Pre-requisites
+
+* Provision a virtual machine that has at least 4GB - 8GB of RAM, 1-2 dedicated CPU cores
+* Install Docker on your virtual machine (pganalyze Enterprise Edition is delivered through a Docker image)
+* PostgreSQL database that can be used for storing statistics data (50 GB of disk space or more)
+* Optional: If you use the RDS integration, you will also need to setup an appropriate IAM role that can be used - you can also assign the required policy to an existing EC2 instance and it's instance role. Appendix A lists out the exact permissions required.
+
+### Step 1: Download the Docker Image
+
+The Docker image is made available to you through the container hosting service quay.io. We have provided you with the credentials you can use for access by email.
+
+Enable access to the image like this:
+
+```
+docker login -e="." -u="pganalyze+enterprise_customer" -p="YOUR_PASSWORD" quay.io
+```
+
+Note that this will add the login information to your current user's `.dockercfg` file - you will need this on every host where you'd like to access the image.
+
+You should now be able to pull the pganalyze Enterprise Edition Docker image:
+
+```
+docker pull quay.io/pganalyze/enterprise:[version]
+```
+
+Please note that `[version]` should be replaced with the version you are using. We recommend using [the latest release](/docs/enterprise/releases).
+
+
+### Step 2: Setting Environment Variables
+
+The Docker image uses environment variables that are passed in to determine how to access outside services, like PostgreSQL, LDAP and mail servers.
+
+Here is an example .env file to get you started:
+
+```
+DATABASE_URL=postgres://myusername:mypassword@myserver.company.net:5432/mydatabase
+MAILER_URL=smtp://myusername:mypassword@mymailserver.company.net:25
+LICENSE_KEY=KEYKEYKEY
+```
+
+You can find the full explanation of all variables in Appendix C.
+
+
+### Step 3: Initialize Database & Verify Installation
+
+First, run the Enterprise self check, to ensure the configuration is correct:
+
+```
+docker run --env-file pganalyze.env quay.io/pganalyze/enterprise:[version] rake enterprise:self_check
+```
+
+```
+Testing database connection... Success!
+Testing Redis connection... Success!
+Testing LDAP connection... Success!
+
+All tests completed successfully!
+```
+
+In the case of an error this will give you a very verbose error message - please don't hesitate to ask us about it.
```

Comment:
I suppose enterprise customers will know how to reach us by this point, but should we be more explicit about a contact mechanism here?